src/app/router.js

```diff
@@ -22,16 +22,44 @@ define([
   'views/layout/footer',
   'views/layout/tutorial',
   'views/modals/splash'
-], function($, deparam, _, Backbone, CityModel, ScorecardModel,
-            CityBuildings, MapView, AddressSearchView,
-            YearControlView, ActivityIndicator,
-            BuildingCounts, CompareBar, ScorecardController,
-            Button, MobileAlert, ModalModel, ModalController, FooterView, TutorialView, SplashView) {
+], function (
+  $,
+  deparam,
+  _,
+  Backbone,
+  CityModel,
+  ScorecardModel,
+  CityBuildings,
+  MapView,
+  AddressSearchView,
+  YearControlView,
+  ActivityIndicator,
+  BuildingCounts,
+  CompareBar,
+  ScorecardController,
+  Button,
+  MobileAlert,
+  ModalModel,
+  ModalController,
+  FooterView,
+  TutorialView,
+  SplashView
+) {
   var RouterState = Backbone.Model.extend({
     queryFields: [
-      'filters', 'categories', 'layer',
-      'metrics', 'sort', 'order', 'lat',
-      'lng', 'zoom', 'building', 'report_active', 'city_report_active'
+      'filters',
+      'categories',
+      'layer',
+      'metrics',
+      'sort',
+      'order',
+      'lat',
+      'lng',
+      'zoom',
+      'building',
+      'report_active',
+      'city_report_active',
+      'tab'
```

Comment:
This, and my note below are the only changes in this file to allow a query param for `tab` in the building report. Sorry for the reformatting here. I can update my prettier config if this is a headache.